### PR TITLE
Add width support to Image

### PR DESCRIPTION
**Pull request by unscsprt**
Addresses #461

### CODE REVIEW

**Comment by unscsprt** [comment]
[IE image responsiveness bug](https://trello.com/c/k7k9P8se/11-ie-image-responsiveness-bug)

File: packages/core/storybook/Image.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { storiesOf } from '@storybook/react'
 import { withInfo } from '@storybook/addon-info'
-import { Image, Box } from '../src'
+import { Image, Box, Flex } from '../src'
```
Comment (James300) [inline packages/core/storybook/Image.js]: Do you need Flex here?

Comment (James300) [reply packages/core/storybook/Image.js]: And you might be able to remove Box too

Comment (unscsprt) [reply packages/core/storybook/Image.js]: Good catch

**Pull request merged**